packages/commonwealth/server/util/entitiesProxy.ts

```diff
@@ -21,8 +33,8 @@ async function ceProxy(ceEndpoint: 'entities' | 'events', req, res) {
 
 function setupCEProxy(app: Express) {
   // using bodyParser here because cosmjs generates text/plain type headers
-  app.get('/api/ce/entities', ceProxy.bind(null, 'entities'));
-  app.get('/api/ce/events', ceProxy.bind(null, 'events'));
+  app.get('/api/ce/entities', calcCeProxyCacheKeyDuration.bind(null, 'entities'), cacheDecorator.cacheMiddleware(60*5, lookupKeyDurationInReq), ceProxy.bind(null, 'entities'));
+  app.get('/api/ce/events', calcCeProxyCacheKeyDuration.bind(null, 'events'), cacheDecorator.cacheMiddleware(60*5, lookupKeyDurationInReq), ceProxy.bind(null, 'events'));
```

Comment:
and this:
https://github.com/hicommonwealth/commonwealth/pull/3803/files#r1191623584